### DISCUSSION TITLE
Cheatsheet TUI (search engine?)

### DISCUSSION THREAD

**Discussion by dlvhdr** [charmbracelet/inspo#21] [Ideas]
A TUI you can search for commands in.
The TUI can then show results from `tldr`, `cheat.sh`, `man` and the command's `--help`.

A one stop shop for command help.

**Comment by zackproser** [comment]
> A TUI you can search for commands in.
> The TUI can then show results from `tldr`, `cheat.sh`, `man` and the command's `--help`.
> 
> A one stop shop for command help.

Nice, I was itching to write something like this using bubbletea and wrapping cheat.sh, but your idea to include other sources is even better.

**Comment by alfiankan** [comment]
how about this
https://github.com/alfiankan/commander and i have a proper one docs here https://alfiankan.github.io/commander-charts/
so we can using charts, charts i predefined command cheatsheet or general snippet available online here https://alfiankan.github.io/commander-charts/charts/ and we can create own chart on local machine than share to the other or contribute to https://alfiankan.github.io/commander-charts/charts/

Spoiler :
![cmdr-ads](https://user-images.githubusercontent.com/40946917/199607983-86460e70-5dc4-474e-abed-14eea38f15d9.gif)


can i add my project to this repo?

**Reply by tomekz** [reply]
> A TUI you can search for commands in. The TUI can then show results from `tldr`, `cheat.sh`, `man` and the command's `--help`.
> 
> A one stop shop for command help.

I could be a fun project 💪
Some time ago I wrote a small bubbletea tui with nested models and fetching data from external API so would be happy to contribute
